lib/walker.js

```diff
@@ -17,12 +18,20 @@ import {
   isDotNODE,
   isPackageJson,
   normalizePath,
+  toNormalizedRealPath,
 } from '../prelude/common';
 
 import { follow, natives } from './follow';
 import { log, wasReported } from './log';
-import * as detector from './detector';
-
+import {
+  detect,
+  visitor_MALFORMED,
+  visitor_NONLITERAL,
+  visitor_USESCWD,
+  visitor_SUCCESSFUL,
+} from './detector';
+
+const strictVerify = Boolean(process.env.PKG_STRICT_VER);
```

Comment:
@erossignon Do you have added something on readme to document this?

Reply:
@robertsLando  the strict strictVerify is to be used by developers of the pkg, not the end-user.
The primary goal is to make sure that we are storing content and state of real files, not symlinks.

The assertion done by strictVerify are quite expensive (following the link) and should not be enabled in production.

I take you point and add a comment at least about this line in the code, so that developers can understand what this is doing.

Reply:
Ok right, but yeah we should document this somewhere as there could be many other devs in future in this project :) 

I'm thinking about a DEVELOPERS.md file where we add some docs to better explain how pkg works on the low level explaining what each file is used for and we could also add this process env var :)

Based on the PR you wrote here I think you have a very complete knowladge of how pkg works, would you like to submit a PR also for that document? I could help you there

Reply:
@robertsLando , please check my latest commit that contains the comment and tell me if it makes sense

Reply:
@robertsLando  => Good point a DEVELOPPERS.md file could be helpfull, ( in a separate PR)